src/util/CMakeLists.txt

```diff
@@ -28,18 +28,18 @@ else()
       config_inc_v REGEX \"CBMC_VERSION *= *[0-9\.]+\")
     string(REGEX REPLACE \"^CBMC_VERSION *= *\" \"\" CBMC_RELEASE \${config_inc_v})
     set(GIT_INFO \"n/a\")
-    configure_file(\${CUR}/version.h.in version.h)
+    configure_file(\${CUR}/version.cpp.in version.cpp)
     "
   )
 endif()
 
-file(WRITE ${CMAKE_CURRENT_BINARY_DIR}/version.h.in
-  "\#define CBMC_VERSION \"@CBMC_RELEASE@ (@GIT_INFO@)\"\n")
+file(WRITE ${CMAKE_CURRENT_BINARY_DIR}/version.cpp.in
+  "extern const char CBMC_VERSION[]=\"@CBMC_RELEASE@ (@GIT_INFO@)\";\n")
```

Comment:
Remove `extern` from the definition, and use `const char *` to match the declaration?

Reply:
The extern is needed, or clang will complain that the symbol is unused.